React/tic-tac-toe/src/utils/localStorage.js

```diff
@@ -0,0 +1,20 @@
+export const loadState = () => {
+  try {
+    const serializedState = sessionStorage.getItem('state');
+    if (serializedState === null) {
+      return undefined;
+    }
+    return JSON.parse(serializedState);
+  } catch (err) {
+    return undefined;
+  }
+};
+
+export const saveState = (state) => {
+  try {
+    const serializedState = JSON.stringify(state);
+    sessionStorage.setItem('state', serializedState);
+  } catch (err) {
```

Comment:
indentation is wrong.. check with eslint and prettier

Reply:
> indentation is wrong.. check with eslint and prettier

it's a 2 spaces tab, eslint and prettier show me no error here :(